.github/workflows/reviewdog.yml

```diff
@@ -8,6 +8,26 @@ permissions:
   checks: write
 
 jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

Comment:
Can we use v4 like https://github.com/otwcode/otwarchive/pull/4627?

Reply:
:facepalm: that was on my mental list of things to do, and I totally forgot. Thanks for reminding me!